src/index.js

```diff
@@ -48,6 +50,7 @@ class Dropzone extends React.Component {
       document.addEventListener('dragover', Dropzone.onDocumentDragOver, false)
       document.addEventListener('drop', this.onDocumentDrop, false)
     }
+    this.fileInputEl.addEventListener('click', this.onInputElementClick, false)
```

Comment:
I'm wondering now why you had to use refs to attache event listener? Was `onClick` in the `inputProps` not enough?